kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -657,6 +657,13 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_SYNC_INIT: ConfigEntry[Boolean] =
+    buildConf("session.engine.sync.init")
+      .doc("When opening kyuubi session, whether to init backend engine synchronously")
```

Comment:
When false, the Kyuubi server will set up the connection with the client without delay as the backend engine will be created asynchronously if it is absent.